monitoring/oplog-populator/dashboard.py

```diff
@@ -182,6 +183,17 @@ class Metrics:
         expr='sum (rate(' + Metrics.CONNECTOR_CONFIG_APPLIED('success="false"') + '))',
     )])
 
+retained_buckets = TimeSeries(
+    title='Retained Buckets',
+    dataSource='${DS_PROMETHEUS}',
+    gradientMode='opacity',
+    lineInterpolation='smooth',
+    legendDisplayMode='hidden',
+    fillOpacity=10,
+    spanNulls=True,
+    targets=[Target(
+        expr='sum(' + Metrics.BUCKETS_RETAINED.raw() + ')',
+    )])
 
```

Comment:
do we really want to _display_ that information?
this should not be a nominal case, just having the alert is probably enough IMHO...